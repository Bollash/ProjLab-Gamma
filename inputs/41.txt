Create Space
Add Asteroid Uran
Mod Counter 0 1
CountDown 0
Status All