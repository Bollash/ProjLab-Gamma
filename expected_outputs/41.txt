Space létrehozva
A nyersanyaggal teli aszteroida hozzá lett adva a space-hez.
A napközeli visszaszámláló a kapott értékre állítódott.
Napközelbe került az aszteroida.
Space
0
10
10
False
1
0
Asteroid
0
3
3
3
Uran
2
0
0
